File: pjecalc-fonte/negocio/br/jus/trt8/pjecalc/negocio/constantes/AbrangenciaDoFeriadoEnum.java
/*
 * Decompiled with CFR 0.152.
 */
package br.jus.trt8.pjecalc.negocio.constantes;

public enum AbrangenciaDoFeriadoEnum {
    FEDERAL("Nacional", "F"),
    ESTADUAL("Estadual", "E"),
    MUNICIPAL("Municipal", "M");

    private String nome;
    private String valor;

    private AbrangenciaDoFeriadoEnum(String nome, String valor) {
        this.nome = nome;
        this.valor = valor;
    }

    public String getNome() {
        return this.nome;
    }

    public String getValor() {
        return this.valor;
    }
}

File: src/lib/pjecalc/core/constantes/abrangencia-do-feriado-enum.ts
/**
 * PJe-Calc v2.15.1 — AbrangenciaDoFeriadoEnum
 * Porte 1:1 de: br.jus.trt8.pjecalc.negocio.constantes.AbrangenciaDoFeriadoEnum
 *
 * Esfera do feriado: nacional (Lei 662/1949), estadual (leis estaduais)
 * ou municipal (leis municipais). Afeta a aplicabilidade ao caso
 * conforme a localidade do reclamante.
 *
 * Ref Java: pjecalc-fonte/negocio/br/jus/trt8/pjecalc/negocio/constantes/AbrangenciaDoFeriadoEnum.java
 */

export enum AbrangenciaDoFeriadoEnum {
  /** Nacional (Lei 662/1949 + Lei 6.802/1980 — Padroeira do Brasil etc). */
  FEDERAL = 'F',
  ESTADUAL = 'E',
  MUNICIPAL = 'M',
}

export const AbrangenciaDoFeriadoEnumNomes: Record<AbrangenciaDoFeriadoEnum, string> = {
  [AbrangenciaDoFeriadoEnum.FEDERAL]: 'Nacional',
  [AbrangenciaDoFeriadoEnum.ESTADUAL]: 'Estadual',
  [AbrangenciaDoFeriadoEnum.MUNICIPAL]: 'Municipal',
};
